Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN apt -qq update && \
       texlive-luatex \
       texlive-pictures \
       texlive-xetex \
-    && rm -rf /var/lib/apt/lists/*
+    && rm -rf /var/lib/apt-get/lists/*
```

Comment:
Is this right? Does `apt-get` really use a different directory than the `apt` command?

Reply:
I'm guessing this is an error:
https://github.com/hadolint/hadolint/wiki/DL3009

> ### Correct code:
> ```
> RUN apt-get update && apt-get install -y python \
>  && apt-get clean \
>  && rm -rf /var/lib/apt/lists/*
> ```

Reply:
Ups, sorry, did not mean to disregard your review. Will not merge it immediately next time.

Reply:
It's fine, I posted this after.

Reply:
No, it doesn't.

```
$ docker run --rm -it matplotlib/mpl-docker:debug-latest /bin/bash -c "apt-get install -qq vim && ls -lhtr /var/lib/apt/lists && ls -lhtr /var/lib/apt-get/lists"
debconf: delaying package configuration, since apt-utils is not installed
Selecting previously unselected package xxd.
(Reading database ... 55102 files and directories currently installed.)
Preparing to unpack .../xxd_2%3a8.1.0875-5_amd64.deb ...
Unpacking xxd (2:8.1.0875-5) ...
Selecting previously unselected package vim-common.
Preparing to unpack .../vim-common_2%3a8.1.0875-5_all.deb ...
Unpacking vim-common (2:8.1.0875-5) ...
Selecting previously unselected package libgpm2:amd64.
Preparing to unpack .../libgpm2_1.20.7-5_amd64.deb ...
Unpacking libgpm2:amd64 (1.20.7-5) ...
Selecting previously unselected package vim-runtime.
Preparing to unpack .../vim-runtime_2%3a8.1.0875-5_all.deb ...
Adding 'diversion of /usr/share/vim/vim81/doc/help.txt to /usr/share/vim/vim81/doc/help.txt.vim-tiny by vim-runtime'
Adding 'diversion of /usr/share/vim/vim81/doc/tags to /usr/share/vim/vim81/doc/tags.vim-tiny by vim-runtime'
Unpacking vim-runtime (2:8.1.0875-5) ...
Selecting previously unselected package vim.
Preparing to unpack .../vim_2%3a8.1.0875-5_amd64.deb ...
Unpacking vim (2:8.1.0875-5) ...
Setting up libgpm2:amd64 (1.20.7-5) ...
Setting up xxd (2:8.1.0875-5) ...
Setting up vim-common (2:8.1.0875-5) ...
Setting up vim-runtime (2:8.1.0875-5) ...
Setting up vim (2:8.1.0875-5) ...
update-alternatives: using /usr/bin/vim.basic to provide /usr/bin/vim (vim) in auto mode
update-alternatives: using /usr/bin/vim.basic to provide /usr/bin/vimdiff (vimdiff) in auto mode
update-alternatives: using /usr/bin/vim.basic to provide /usr/bin/rvim (rvim) in auto mode
update-alternatives: using /usr/bin/vim.basic to provide /usr/bin/rview (rview) in auto mode
update-alternatives: using /usr/bin/vim.basic to provide /usr/bin/vi (vi) in auto mode
update-alternatives: using /usr/bin/vim.basic to provide /usr/bin/view (view) in auto mode
update-alternatives: using /usr/bin/vim.basic to provide /usr/bin/ex (ex) in auto mode
update-alternatives: using /usr/bin/vim.basic to provide /usr/bin/editor (editor) in auto mode
Processing triggers for hicolor-icon-theme (0.17-2) ...
Processing triggers for libc-bin (2.28-10) ...
Processing triggers for mime-support (3.62) ...
total 17M
-rw-r--r-- 1 root root  17M May  9 08:56 deb.debian.org_debian_dists_buster_main_binary-amd64_Packages.lz4
-rw-r--r-- 1 root root 119K May  9 10:04 deb.debian.org_debian_dists_buster_InRelease
-rw-r--r-- 1 root root  13K Jun 13 19:56 deb.debian.org_debian_dists_buster-updates_main_binary-amd64_Packages.lz4
-rw-r--r-- 1 root root 393K Jul  8 20:46 security.debian.org_debian-security_dists_buster_updates_main_binary-amd64_Packages.lz4
-rw-r--r-- 1 root root  64K Jul 11 19:22 security.debian.org_debian-security_dists_buster_updates_InRelease
-rw-r--r-- 1 root root  51K Jul 11 20:27 deb.debian.org_debian_dists_buster-updates_InRelease
-rw-r----- 1 root root    0 Jul 11 22:35 lock
drwxr-xr-x 1 _apt root 4.0K Jul 11 22:35 auxfiles
drwx------ 2 _apt root 4.0K Jul 11 22:35 partial
ls: cannot access '/var/lib/apt-get/lists': No such file or directory
```

I didn't know this until checking now, as I've used this pattern in all my Dockerfile and never had it yell at me. Thanks for the catch @QuLogic and I'll open a PR to revert this now.

Reply:
I tried to run it interactivaly too, saw `/var/lib/apt` only dir after the install of `ffmpeg`:
```
(on my machine) $ docker run -it --rm python:3.8 bash
root@eaa5f6a555e3:/# apt-get -qq update && \
>     apt-get install --no-install-recommends -y ffmpeg
...
root@eaa5f6a555e3:/var/lib/apt/lists# ls -al
total 17068
drwxr-xr-x 1 root root     4096 Jul 27 20:54 .
drwxr-xr-x 1 root root     4096 Jul 27 20:54 ..
drwxr-xr-x 2 _apt root     4096 Jul 27 20:54 auxfiles
-rw-r--r-- 1 root root    51901 Jul 27 20:19 deb.debian.org_debian_dists_buster-updates_InRelease
-rw-r--r-- 1 root root    13206 Jun 13 19:56 deb.debian.org_debian_dists_buster-updates_main_binary-amd64_Packages.lz4
-rw-r--r-- 1 root root   121474 May  9 10:04 deb.debian.org_debian_dists_buster_InRelease
-rw-r--r-- 1 root root 16782044 May  9 08:56 deb.debian.org_debian_dists_buster_main_binary-amd64_Packages.lz4
-rw-r----- 1 root root        0 Jul 27 20:54 lock
drwx------ 2 _apt root     4096 Jul 27 20:54 partial
-rw-r--r-- 1 root root    65372 Jul 27 14:01 security.debian.org_debian-security_dists_buster_updates_InRelease
-rw-r--r-- 1 root root   411406 Jul 26 19:57 security.debian.org_debian-security_dists_buster_updates_main_binary-amd64_Packages.lz4
```